normalizer/tests/test_all.py

```
@@ -0,0 +1,147 @@
+            print(f"{test_name}: [SKIP]")
+        elif result == True:
+            print(f"{test_name}: [PASS]")
+        else:
+            print(f"{test_name}: [FAIL]")
+    print("-" * 60)
+    
+    # pytest用の最終アサーション
+    assert all_passed, f"Some tests failed. Results: {test_results}"
+    
+    if all_passed:
+        print("[SUCCESS] すべてのテストが成功しました！")
```

Ruff (0.14.13)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)